js/script.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const levels = document.querySelector('.levels');
+let levelTable = document.querySelectorAll('.level');
+let tableNumber = 0;
+levelTable = [].slice.call(levelTable);
+
+levels.addEventListener('click', (e) => {
+  if (e.target && e.target.classList.contains('level')) {
+    tableNumber = levelTable.indexOf(event.target);
+    levelTable[tableNumber].classList.add('selected-level', 'lvl-active');
+    
+    levelTable.forEach((item, i) => {
+      if (i !== tableNumber) {
+        item.classList.remove('selected-level', 'lvl-active');
+      }
+    });
+  }
+});
+
+const start = document.querySelector('.play');
+start.addEventListener('click', render);
```

Comment:
Codacy found an issue: ['render' was used before it was defined.](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)